IdentityServer/Quickstart/TestUsers.cs

```diff
@@ -71,6 +71,50 @@ public class TestUsers
+                                                         new Claim(JwtClaimTypes.Email, "jerrySmith@email.com"),
+                                                         new Claim(JwtClaimTypes.EmailVerified,
+                                                                   "true",
+                                                                   ClaimValueTypes.Boolean),
+                                                         new Claim(JwtClaimTypes.WebSite, "http://jerry.com"),
+                                                         new Claim(JwtClaimTypes.Address,
+                                                                   @"{ 'street_address': 'One Hacker Way', 'locality': 'Heidelberg', 'postal_code': 69118, 'country': 'Germany' }",
+                                                                   IdentityServerConstants.ClaimValueTypes.Json),
+                                                         new Claim(JwtClaimTypes.Scope,
+                                                                   nameof(Defaults.Scopes.HighlightWrite))
+                                                     }
+                                                 },
+                                                 new TestUser
+                                                 {
+                                                     SubjectId = "147852369",
+                                                     Username = "berry",
+                                                     Password = "berry",
+                                                     Claims =
+                                                     {
+                                                         new Claim(JwtClaimTypes.Name, "berry Smith"),
+                                                         new Claim(JwtClaimTypes.GivenName, "berry"),
+                                                         new Claim(JwtClaimTypes.FamilyName, "Smith"),
+                                                         new Claim(JwtClaimTypes.Email, "berrySmith@email.com"),
+                                                         new Claim(JwtClaimTypes.EmailVerified,
+                                                                   "true",
+                                                                   ClaimValueTypes.Boolean),
+                                                         new Claim(JwtClaimTypes.WebSite, "http://berry.com"),
+                                                         new Claim(JwtClaimTypes.Address,
+                                                                   @"{ 'street_address': 'One Hacker Way', 'locality': 'Heidelberg', 'postal_code': 69118, 'country': 'Germany' }",
+                                                                   IdentityServerConstants.ClaimValueTypes.Json),
+                                                         new Claim(JwtClaimTypes.Scope,
```

Comment:
Does this even do anything? Why do they need highlight write? ^^ These might have been left overs from my old implementation